docs/usage.md

```diff
@@ -425,6 +425,56 @@ field_name_mapping = {
 ```
 
 
+### Working with polymorphic resources
+
+Polymorphic resources (as backed by [django-polymorphic](https://django-polymorphic.readthedocs.io/en/stable/))
+allow you to easily use specialized subclasses without have to have special
+endpoints to expose the specialized versions. For example, if you had a
+`Project` that could be either an `ArtProject` or a `ResearchProject`, you can
+have both kinds at the same URL.
+
```

Comment:
I think we could put a new paragraph here that includes comments about polymorphic libraries (and remove the "as backed by" parenthetical in the previous paragraph). How does the following sound?

> DJA tests its polymorphic support against [django-polymorphic](https://django-polymorphic.readthedocs.io/en/stable/). The polymorphic feature should also work with other popular libraries like `django-polymodels` or `django-typed-models`.